README.adoc

```diff
@@ -82,15 +82,21 @@ DHCP network is added by default and a warning is displayed.
 1.  Run the plugin to get environment variables and certificates:
 +
 ----------------------------------------------------------------------------
-$ vagrant service-manager env docker
-# Set the following environment variables to enable access to the
-# docker daemon running inside of the vagrant virtual machine:
-export DOCKER_HOST=tcp://172.28.128.4:2376
-export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
-export DOCKER_TLS_VERIFY=1
-export DOCKER_API_VERSION=1.21
-# run following command to configure your shell:
-# eval "$(vagrant service-manager env docker)"
+ # docker env:
```

Comment:
Please use following instead :
```
$ vagrant service-manager env
# docker env:
# Set the following environment variables to enable access to the
# docker daemon running inside of the vagrant virtual machine:
export DOCKER_HOST=tcp://172.28.128.182:2376
export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
export DOCKER_TLS_VERIFY=1
export DOCKER_API_VERSION=1.20

# run following command to configure your shell:
# eval "$(vagrant service-manager env docker)"
```

Since `docker` is default in ADB, we should only show docker environment variables. Showing Openshift environment variables might confuse user as it a customized case and can be achieved using Openshift Vagrantfile.